adapters/tlsio_openssl.c

```diff
@@ -1238,7 +1238,7 @@ void tlsio_openssl_deinit(void)
 {
     openssl_dynamic_locks_uninstall();
     openssl_static_locks_uninstall();
-#if  (OPENSSL_VERSION_NUMBER >= 0x00907000L) &&  (OPENSSL_VERSION_NUMBER < 0x20000000L) && (FIPS_mode_set)
+#if  (OPENSSL_VERSION_NUMBER >= 0x00907000L) && (FIPS_mode_set)
```

Comment:
Should we just change the check to < 0x30000000L ?

Reply:
Just to make sure.

Reply:
You know, so we get a notification somehow if a new version comes out. Gives us the chance to check it out and make sure it works.

Reply:
Same comment in all the checks that were modified.

Reply:
We could but I would say we keep it removed until there comes a version that actually removes these APIs. At which point we can set the guard on that version. Otherwise it's a constantly moving goal post.